namerd/dcos-bootstrap/src/main/scala/io/buoyant/namerd/DcosBootstrap.scala

```diff
@@ -0,0 +1,60 @@
+package io.buoyant.namerd
+
+import com.twitter.finagle.Dtab
+import com.twitter.finagle.serverset2.ZkClient
+import com.twitter.io.Buf
+import io.buoyant.admin.App
+import java.io.File
+import scala.io.Source
+
+object Main extends App {
+
+  val defaultNs = "default"
+  val defaultDtab = Dtab.read(
+    """|/srv        => /io.l5d.marathon;
+       |/www        => /srv/web;
+       |/host       => /srv | /$/io.buoyant.http.anyHostPfx/www;
+       |/method     => /$/io.buoyant.http.anyMethodPfx/host;
+       |/http/1.1   => /method;
+       |/http/1.0   => /$/io.buoyant.http.anyMethodPfx/www;
+       |""".stripMargin
+  )
+
+  def main(): Unit = {
+    args match {
+      case Array(path) =>
+        val config = loadNamerd(path)
```

Comment:
[Q] did you decide to go with the config parsing over just trying to extract the information you needed from a `Map[String, Any]`?  Needing to depend on every possible plugin is not great.